cpg_workflows/resources.py

```diff
@@ -29,7 +29,7 @@ class MachineType:
     """
 
     min_cpu: int = 2
-    threads_on_cpu = 2  # hyper-threading
+    threads_on_cpu = 1  # GCP does not have hyper-threading
```

Comment:
The documentation linked to is not incredibly clear, but I think it is saying that the vCPU count is a count of hyper-threads, not a count of cores (= 2 hyper-threads). It goes on to say that two vCPUs will share a physical core by default (thus GCP does have hyper-threading), but you can switch that off (to maybe get performance/security/licensing benefits) but then you'll still be charged for each whole core even though you're only using one hyper-thread apiece.

So the comment is a bit misleading and might be better as something like “GCP vCPUs correspond to hyper-threads, not physical cores”.

Reply:
Good point. I made it more explicit.